### PR TITLE
Added Eyes Detection

### LINKED ISSUE #629: Eye detection
I want to add a eye detection feature.
please assign me for this issue

### PR DESCRIPTION
**Pull request by Avishake007**
# Description

It will detect your eyes and will count the number of eyes 

## Have you read the [Contributing Guidelines on Pull Requests](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/CONTRIBUTING.md)?

- [x] Yes
- [ ] No

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] My documentation follows [`Template for README.md`](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Template%20for%20README.md)
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works.
eyes(2)

### CODE REVIEW

File: Eyes Detection/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+
+<h1 align="center"> Body Detection</h1>
```
Comment (santushtisharma10) [inline Eyes Detection/README.md:2]: @Avishake007 change the heading to Eye Detection

File: Eyes Detection/eyes.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        # Draw rectangle around the eyes
+    for (x, y, w, h) in eyes:
+        cv2.rectangle(color_image, (x, y), (x + w, y + h), (0, 255, 0), 2)
+        
+        # Show image
+    cv2.imshow('Image', color_image)
+    cv2.imwrite('detect.png',color_image)
+    if cv2.waitKey(1) &  0xFF == ord('q'):
+        break
+
```
Comment (santushtisharma10) [inline Eyes Detection/eyes.py:33]: @Avishake007  add `capture.release()`

**Review by santushtisharma10 — CHANGES_REQUESTED** [review]

**Comment by Avishake007** [comment]
Sure, I will change it and commit again

File: Eye Detection/eyes.py [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+capture=cv2.VideoCapture(0) #To open camera 1
+
+if (capture.isOpened()==False):
+	print("Sorry")
+while(capture.isOpened()):
+
+        # Load trained cascade classifier
+    ret,color_image=capture.read()
+    eye_cascade=cv2.CascadeClassifier('haarcascade_eye.xml')
```
Comment (kaustubhgupta) [inline Eye Detection/eyes.py:13]:
```suggestion
    eye_cascade=cv2.CascadeClassifier('./Eye Detection/haarcascade_eye.xml')
```

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]
Just one minor change:

File: Eye Detection/eyes.py [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+capture=cv2.VideoCapture(0) #To open camera 1
+
+if (capture.isOpened()==False):
+	print("Sorry")
+while(capture.isOpened()):
+
+        # Load trained cascade classifier
+    ret,color_image=capture.read()
+    eye_cascade=cv2.CascadeClassifier('../Eye Detection/haarcascade_eye.xml')
```
Comment (kaustubhgupta) [inline Eye Detection/eyes.py:13]:
```suggestion
    eye_cascade=cv2.CascadeClassifier('./Eye Detection/haarcascade_eye.xml')
```

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

**Comment by Avishake007** [comment]
@kaustubhgupta  when I am making **eye_cascade=cv2.CascadeClassifier('./Eye Detection/haarcascade_eye.xml')** change in my code its giving an error 
eyes(3)
 but when I am keeping my files and folder structure like this 
eyes(4)
its working. So will I keep the cascade file according to this format

**Comment by kaustubhgupta** [comment]
> @kaustubhgupta when I am making **eye_cascade=cv2.CascadeClassifier('./Eye Detection/haarcascade_eye.xml')** change in my code its giving an error
> eyes(3)
> but when I am keeping my files and folder structure like this
> eyes(4)
> its working. So will I keep the cascade file according to this format and then make the necessary changes

As you have the same folder named Eye Detection storing another folder Eye Detection. then according to hierarchy, it is referring to the upper level folder. You can commit my suggestion and it will work everywhere

**Comment by Avishake007** [comment]
@kaustubhgupta  Its done Kindly check

**Review by kaustubhgupta — APPROVED** [review]
@Avishake007 💯

**Review by santushtisharma10 — APPROVED** [review]
@Avishake007 💯

**Comment by kaustubhgupta** [comment]
@antrikshmisri

**Review by antrikshmisri — APPROVED** [review]

**Review by avinashkranjan — APPROVED** [review]

**Pull request merged**